app/components/projects/settings/life_cycles/step_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<%=
+  flex_layout(align_items: :center,
+              justify_content: :space_between) do |step_container|
+    step_container.with_column(flex_layout: true) do |title_container|
+      title_container.with_column(pt: 1, mr: 3) do
+        render(Primer::Beta::Text.new(classes: 'filter-target-visible-text')) { definition.name }
```

Comment:
Removes ability to filter by step type, not sure such unintentional feature should be there or not

Reply:
It can be an option to mark also type element with the class and handle multiple, but probably more correct to not support this unintentional feature